diff --git a/configure b/configure
index a0eed36..1f1e0f6 100755
--- a/configure
+++ b/configure
@@ -3512,6 +3512,7 @@ vp9_vaapi_encoder_select="vaapi_encode"
 vp9_qsv_encoder_deps="libmfx MFX_CODEC_VP9"
 vp9_qsv_encoder_select="qsvenc"
 vp9_v4l2m2m_decoder_deps="v4l2_m2m vp9_v4l2_m2m"
+vp9_v4l2m2m_encoder_deps="v4l2_m2m vp9_v4l2_m2m"
 vvc_qsv_decoder_select="vvc_mp4toannexb_bsf qsvdec"
 
 # parsers
